clusterloader2/api/validation/validation.go

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"fmt"
+
+	"k8s.io/perf-tests/clusterloader2/api"
+)
+
+// VerifyConfig does simple validatio for config and all included structures.
+// Returns error is any condition is not satisfied.
+func VerifyConfig(c *api.Config) error {
+	if c.AutomanagedNamespaces < 1 {
+		return fmt.Errorf("Expected one or more namespaces!")
```

Comment:
Not necessary  - I can imagine scenarios where users don't want to have any automanaged scenarios at all and want to handle everything themselves.

Reply:
Changed to >= 0 required.